packages/block-library/src/navigation/index.php

```diff
@@ -463,15 +468,14 @@ function block_core_navigation_get_default_pages_fallback() {
 }
 
 /**
- * Retrieves the appropriate fallback to be used on the front of the
- * site when there is no menu assigned to the Nav block.
- *
- * This aims to mirror how the fallback mechanic for wp_nav_menu works.
- * See https://developer.wordpress.org/reference/functions/wp_nav_menu/#more-information.
- *
- * @return array the array of blocks to be used as a fallback.
+ * Creates the appropriate fallback to be used on the front of the site.
  */
-function block_core_navigation_get_fallback_blocks() {
+function block_core_navigation_create_fallback() {
+	$should_skip = apply_filters( 'block_core_navigation_skip_fallback', false );
```

Comment:
This allows consumers to opt out of the auto-creation of a Nav post behaviour.

If they choose to do so then the block will always start empty.

Reply:
phpdocs would be useful here.

Reply:
This was my fault. I introduced a `return` where one was not needed. I've annotated anyway and also improved the docblock.